### ISSUE TITLE
Not currently working

### ISSUE THREAD

**Issue by johtso** [jlubcke/pytest-pycharm#1]
```
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/main.py", line 84, in wrap_session
INTERNALERROR>     doit(config, session)
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/main.py", line 122, in _main
INTERNALERROR>     config.hook.pytest_runtestloop(session=session)
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/core.py", line 413, in __call__
INTERNALERROR>     return self._docall(methods, kwargs)
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/core.py", line 424, in _docall
INTERNALERROR>     res = mc.execute()
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/core.py", line 315, in execute
INTERNALERROR>     res = method(**kwargs)
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/main.py", line 142, in pytest_runtestloop
INTERNALERROR>     item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/core.py", line 413, in __call__
INTERNALERROR>     return self._docall(methods, kwargs)
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/core.py", line 424, in _docall
INTERNALERROR>     res = mc.execute()
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/core.py", line 315, in execute
INTERNALERROR>     res = method(**kwargs)
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/pytest_timeout.py", line 72, in pytest_runtest_protocol
INTERNALERROR>     return __multicall__.execute()
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/core.py", line 315, in execute
INTERNALERROR>     res = method(**kwargs)
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/runner.py", line 65, in pytest_runtest_protocol
INTERNALERROR>     runtestprotocol(item, nextitem=nextitem)
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/runner.py", line 75, in runtestprotocol
INTERNALERROR>     reports.append(call_and_report(item, "call", log))
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/runner.py", line 115, in call_and_report
INTERNALERROR>     hook.pytest_exception_interact(node=item, call=call, report=report)
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/main.py", line 167, in call_matching_hooks
INTERNALERROR>     return hookmethod.pcall(plugins, **kwargs)
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/core.py", line 417, in pcall
INTERNALERROR>     return self._docall(methods, kwargs)
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/core.py", line 424, in _docall
INTERNALERROR>     res = mc.execute()
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/_pytest/core.py", line 315, in execute
INTERNALERROR>     res = method(**kwargs)
INTERNALERROR>   File "/Users/human/.virtualenvs/foo/lib/python2.7/site-packages/pytest_pycharm.py", line 27, in pytest_exception_interact
INTERNALERROR>     debugger.force_post_mortem_stop += 1
INTERNALERROR> AttributeError: PyDB instance has no attribute 'force_post_mortem_stop'
```

pytest==2.6.3
pytest-pycharm==0.1.0

**Comment by johtso** [comment]
Yep, `4.0.0`

**Comment by jlubcke** [comment]
I think I have fixed it in pytest-pycharm 0.2.0
